plugins/uefi/uefi.conf

```diff
@@ -10,3 +10,7 @@
 
 # amount of free space required on the ESP, for example using 0x2000000 for 32Mb
 #RequireESPFreeSpace=
+
+# with the UEFI removable path enabled, the default esp path is set to /EFI/boot
+# the shim EFI bianry needs to be found at /EFI/boot/bootx64.efi
```

Comment:
**binary** and presumably this is `$ESP/EFI/boot/bootx64.efi`?

Reply:
Sure. will be fixed.